examples/knative/powercli/kn-pcli-pg-check/test/test-payload.json

```diff
@@ -0,0 +1,158 @@
+{
+	"Key": 689459,
+	"ChainId": 689456,
+	"CreatedTime": "2022-02-24T20:01:17.280999Z",
+	"UserName": "VSPHERE.LOCAL\\Administrator",
+	"Datacenter": {
+	  "Name": "HomeLab",
+	  "Datacenter": {
+		"Type": "Datacenter",
+		"Value": "datacenter-47"
+	  }
+	},
+	"ComputeResource": {
+	  "Name": "CL1",
+	  "ComputeResource": {
+		"Type": "ClusterComputeResource",
+		"Value": "domain-c52"
+	  }
+	},
+	"Host": {
+	  "Name": "esx02.ad.patrickkremer.com",
+	  "Host": {
+		"Type": "HostSystem",
+		"Value": "host-58"
+	  }
+	},
+	"Vm": {
+	  "Name": "REPLACE-ME",
+	  "Vm": {
+		"Type": "VirtualMachine",
+		"Value": "REPLACE-ME"
+	  }
+	},
+	"Ds": null,
+	"Net": null,
+	"Dvs": null,
+	"FullFormattedMessage": "Reconfigured tinycore-2 on esx02.ad.patrickkremer.com in HomeLab.  \n \nModified:  \n \nconfig.hardware.device(4000).backing.port.portgroupKey: \"dvportgroup-1006\" -> \"dvportgroup-8004\"; \n\nconfig.hardware.device(4000).backing.port.portKey: \"4\" -> \"65\"; \n\nconfig.hardware.device(4000).backing.port.connectionCookie: 174899823 -> 193390024; \n\n Added:  \n \n Deleted:  \n \n",
```

Comment:
nit: example text should not contain user-specific data

Reply:
Replaced